python_modules/libraries/dagster-sdf/dagster_sdf/sdf_cli_event.py

```diff
@@ -74,7 +75,7 @@ def to_default_asset_events(
             return
 
         selected_output_names: AbstractSet[str] = (
-            context.selected_output_names if context else set()
+            context.op_execution_context.selected_output_names if context else set()
```

Comment:
`AssetExecutionContext` raises a deprecation warning if `context.selected_output_names` is called, but allows you to get the underlying `op_execution_context`. `OpExecutionContext` also implements `op_execution_context`